daisy_workflows/image_import/windows/run_startup_scripts.cmd

```diff
@@ -16,12 +16,28 @@ REM limitations under the License.
 REM Give the network time to initialize.
 ping 127.0.0.1 -n 60
 
-echo "Translate: Starting image translate..." > COM1:
+set image_import_reg_path="HKLM\Software\ImageImport"
 
-REM Restart the system in 5 minutes. Translate.ps1 will cancel this restart when it starts.
+for /f "tokens=3" %%a in ('reg query %image_import_reg_path% /v StartupScriptRunsCount') do set startup_script_runs_count=%%a
```

Comment:
Have you looked into creating an e2e test that fails prior to this fix, but passes after the fix is applied?

Reply:
Discussed verbally and agreed to test this fix manually only because adding an E2E test for this case sounds tricky.